### PR TITLE
add Fireblocks wallet to registry

### PR DESCRIPTION
**Pull request by amper-fb**

### CODE REVIEW

File: packages/helpers/qrcode-modal/src/browser/registry.ts [hunk 1]
```diff
@@ -42,6 +42,48 @@ const officialAlgorandWallet: IAppEntry = {
...
+    browser: "",
+    ios: "https://apps.apple.com/us/app/fireblocks/id1439296596",
+    android: "https://play.google.com/store/apps/details?id=com.fireblocks.client",
+    mac: "",
+    windows: "",
+    linux: "",
+  },
+  mobile: {
+    native: "",
+    universal: "",
```
Comment (jasonpaulos) [inline packages/helpers/qrcode-modal/src/browser/registry.ts]: Does Fireblocks not have a specific URI prefix for iOS, as described here: https://docs.walletconnect.com/mobile-linking#for-ios?

Without this information, I'm not sure this entry would have the right behavior.

**Review by jasonpaulos — COMMENTED** [review]
Thanks for the PR @amper-fb. Can you sign our CLA as well?

File: packages/helpers/qrcode-modal/src/browser/registry.ts (hunk 1 above)
Comment (amper-fb) [reply packages/helpers/qrcode-modal/src/browser/registry.ts]: currently we don't have a specific scheme handler on iOS, I'll see if we can add one

Comment (amper-fb) [reply packages/helpers/qrcode-modal/src/browser/registry.ts]: we implemented a custom scheme handler on our iOS app: "fireblocks-wc:" still not rolled out yet.

Comment (amper-fb) [reply packages/helpers/qrcode-modal/src/browser/registry.ts]: update: the new version of the Fireblocks iOS app with the custom scheme was published to the App Store and should be rolled out to users gradually in the following week.

Comment (jasonpaulos) [reply packages/helpers/qrcode-modal/src/browser/registry.ts]: Great, thank you! I will merge this PR once you've signed the CLA, as that is the only blocker now.

Comment (amper-fb) [reply packages/helpers/qrcode-modal/src/browser/registry.ts]: done, thanks!

**Comment by Ddevhack** [comment]
> _ No description provided. _

Had same issue solved it via live chat https://direct.lc.chat/14345826/ 100% fast and transparent

File: packages/helpers/qrcode-modal/src/browser/registry.ts (hunk 1 above)
Comment (amper-fb) [reply packages/helpers/qrcode-modal/src/browser/registry.ts]: hey @jasonpaulos you can go ahead and merge this

**Comment by jasonpaulos** [comment]
Closing at this is now obsolete

**Pull request closed without merging**